src/cron.ts

```diff
@@ -183,13 +208,18 @@ export const cron = async (startDate: Date, owner: string, repo: string, octokit
       }
 
       // if we don't find a match, we search for a closed referenda
-      const completedMatch = completedReferendas.find(
+      const completedMatch = finishedReferendas.find(
         ({ hash }) => hash === tx.method.hash.toHex() || hash === tx.method.toHex(),
       );
       if (completedMatch) {
-        logger.info(`Found completed referenda for PR #${pr}`);
+        logger.info(`Found finished referenda for PR #${pr} with state ${completedMatch.state}`);
         const command = `/rfc process ${completedMatch.executedHash}`;
-        const msg = "PR can be merged. Write the following command to trigger the bot" + `\n\n\`${command}\``;
+        const msg =
+          (completedMatch.state === "Executed"
+            ? "PR can be merged."
+            : `Referenda state is \`${completedMatch.state}\`. PR can be closed.`) +
+          "\n\nWrite the following command to trigger the bot\n\n" +
```

Comment:
i guess it makes sense to offer the `command` only if it's executed.. if it's not, then just that message you have about PR to close

Reply:
From what @rzadp said, it can still `close` the PR (not merge) if it's rejected.

Reply:
Wait, so this make this PR redundant? I can't do anything but comment that it is useless now?

Should I even comment on it?

Reply:
> approved (that is, confirmed and executed)

And here, I was accepting just the `confirmed` part - execution is not that important here is it's just an equivalent of a `console.log` (`system.remark`) the way I understand it.

Reply:
> Wait, so this make this PR redundant? I can't do anything but comment that it is useless now?
> Should I even comment on it?

I think it's not redundant, I think we should comment (we comment that a referendum was opened, and at the end it'd be nice to comment what was its result.

Again the process:

- RFC_APPROVE referendum:
  - approved:
    - PR should be merged - `/rfc process` does that.
    - PR users should be notified (existing functionality in cron).
  - rejected/timedout/killed:
    - Don't close/merge the PR.
    - PR users should be notified (this PR can do this)
- RFC_REJECT referendum is symmetrical (I thought your PR is gonna be about it):
  - approved:
    - PR should be closed - `/rfc process` does that.
    - PR users should be notified (missing functionality).
  - rejected/timedout/killed:
    - Don't close/merge the PR.
    - PR users should be notified (missing functionality)

Reply:
Makes sense! I'll do that

Reply:
I have applied the fix.

By the way, the logic is that [if a RFC has been rejected, it closes the PR on the `process` command](https://github.com/paritytech/rfc-action/blob/main/src/process.ts#L82)

Reply:
No, look again at the `referendumState`:

https://github.com/paritytech/rfc-action/blob/4b7d2a911220b32f2222e083fd32aa62f17739c4/src/find-referendum-state.ts#L13

I didn't manage to make it less confusing :D

Reply:
https://github.com/paritytech/rfc-action/blob/4b7d2a911220b32f2222e083fd32aa62f17739c4/src/find-referendum-state.ts#L20

To make it less confusing, the return type should rather be something like:

```
null | "RFC_APPROVE_Approved" | "RFC_REJECT_Approved"
```

Reply:
Gotcha. So I simply remove the message from the non confirmed proposal? Does my latest change works?

Or am I missing something else?

Reply:
Yes, it makes sense to me.